eos-updater-flatpak-installer/main.c

```diff
@@ -0,0 +1,316 @@
+    }
+
+  return TRUE;
+}
+
+static gboolean
+check_if_disabled_on_this_boot (gboolean *disabled, GError **error)
+{
+  gchar *contents = NULL;
+
+  g_return_val_if_fail (disabled != NULL, FALSE);
+
+  if (!g_file_get_contents ("/proc/cmdline", &contents, NULL, error))
+    return FALSE;
+
+  *disabled = g_strrstr (contents, "eos-updater-disable=1") != NULL;
+
+  return TRUE;
+}
+
+static int
+usage (GOptionContext *context,
+       const gchar    *error_message,
+       ...) G_GNUC_PRINTF (2, 3);
+
+static int
+usage (GOptionContext *context,
+       const gchar    *error_message,
+       ...)
+{
+  va_list ap;
+  g_autofree gchar *formatted_message = NULL;
+  g_autofree gchar *help = NULL;
+
+  /* Format the arguments. */
+  va_start (ap, error_message);
+  formatted_message = g_strdup_vprintf (error_message, ap);
+  va_end (ap);
+
+  /* Include the usage. */
+  help = g_option_context_get_help (context, TRUE, NULL);
+  g_printerr ("%s: %s\n\n%s\n", g_get_prgname (), formatted_message, help);
+
+  return 128;
+}
+
+static GFile *
+determine_flatpaks_to_export_file (const gchar *pending_flatpak_deployments_state,
+                                   const gchar *opt_ostree_checksum,
+                                   GError      **error)
+{
+  g_autofree gchar *checksum = NULL;
+  g_autofree gchar *to_export_path = NULL;
+
+  /* If the ostree checksum was already provided to us, then just use that */
+  if (opt_ostree_checksum)
+    {
+      checksum = g_strdup (opt_ostree_checksum);
+    }
+  else
+    {
+      /* Otherwise we need to figure out the checksum from the booted deployment */
+      g_autoptr(OstreeSysroot) sysroot = ostree_sysroot_new_default ();
+      g_autoptr(OstreeDeployment) deployment = NULL;
+
+      if (!ostree_sysroot_load (sysroot, NULL, error))
+        return NULL;
+
+      deployment = ostree_sysroot_get_booted_deployment (sysroot);
+
+      if (!deployment)
+        {
+          g_set_error (error, G_IO_ERROR, G_IO_ERROR_FAILED, "could not get booted deployment");
+          return NULL;
+        }
+
+      checksum = g_strdup (ostree_deployment_get_csum (deployment));
+    }
+
+  to_export_path =  g_build_filename (pending_flatpak_deployments_state,
+                                      checksum,
+                                      "to-export",
+                                      NULL);
+  return g_file_new_for_path (to_export_path);
+}
+
+int
+main (int    argc,
+      char **argv)
+{
+  g_autoptr(GError) error = NULL;
+  g_autoptr(GPtrArray) flatpaks_to_deploy = NULL;
+  g_autoptr(GOptionContext) context = NULL;
+  g_autofree gchar *flatpaks_to_export_path = g_build_filename (pending_flatpak_deployments_state_path (),
+                                                               "to-export",
+                                                               NULL);
+  g_autofree gchar *ostree_checksum = NULL;
+  g_autoptr(GFile) flatpaks_to_export_file = g_file_new_for_path (flatpaks_to_export_path);
```

Comment:
It looks like the value of this variable is never used, and overwritten later by the return value of `determine_flatpaks_to_export_file()`. `flatpaks_to_export_path` will also be unused when this assignment is removed.

Reply:
Indeed, this was a leftover